src/main/java/com/esri/samples/ogc/wmts_layer/README.md

```diff
@@ -0,0 +1,28 @@
+<h1>WMTS Layer</h1>
+
+<p>Shows how to display a WMTS layer from a WMTS service.</p>
```

Comment:
Might replace first WMTS  with `Web Map Tile Service ` so people know what it stands for.